pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "Microsoft.NETCore.Portable.Compatibility": "1.0.2",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.0",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.1-servicing-24720-01",
```

Comment:
Actually I think this is incorrect. We have to be careful with which CoreCLR package we reference for UWP. Actually it looks like a lot of the packages referenced here shouldn't have been updated. At this point we having shipped this from the 1.1 branch so it hasn't caused trouble but if we are to ship it I think there maybe some problems. 

cc @yizhang82